### DISCUSSION TITLE
New release?

### DISCUSSION THREAD

**Discussion by nnerik** [FreeOpcUa/opcua-asyncio#991] [General]
My project is failing because of a bug in the latest release. However, this was resolved in master a few weeks ago, so would it be possible to get a new release? 🙏

**Comment by jrast** [comment]
Since the last release was in May and the project was quite active in the meantime, I would also apreciate a new release!

I also observed a couple of issues / discussions which are solved on master but not in the last release. So releasing a new version would lower the "noise" in new issues.

**Comment by oroulet** [comment]
done

**Reply by jrast** [reply]
Thanks!
